components/nif/niffile.hpp

```diff
@@ -79,11 +79,7 @@ class NIFFile final : public File
     enum NIFVersion
     {
         VER_MW         = 0x04000002,    // 4.0.0.2. Main Morrowind NIF version.
-        VER_CI         = 0x04020200,    // 4.2.2.0. Main Culpa Innata NIF version, also used in Civ4.
-        VER_ZT2        = 0x0A000100,    // 10.0.1.0. Main Zoo Tycoon 2 NIF version, also used in Oblivion and Civ4.
```

Comment:
Says this is used in Oblivion

Reply:
Oblivion can load many things, doesn't mean I should document every single version it supports.